entry_tasks/16_Pick_movie_by _genre_or_actor/main.py

```diff
@@ -0,0 +1,76 @@
+if __name__ == "__main__":
+    # GENRES = []
+    a = input("Start searching by genre: ").strip()
+    genres_data = {}
+    genres_data['comedy'] = ['Meet the Parents', 'Anger Management']
+    genres_data['adventures'] = ['Mummy']
+    genres_data['romantic'] = ['Vanilla Sky', 'Meet Joe Black']
+    genres_data['drama'] = ['Meet Joe Black']
+    genres_data['thriller'] = ['Vanilla Sky']
+    genres_data['action'] = ['Mission Impossible']
+    genres_keys = list(genres_data.keys())
+    gk = f'{">>> Available Genres: "} {genres_keys}'
+    print(gk)
```

Comment:
Don't need to create `gk`, just print formatted string directly.
As a developer you don't want to create a lot of one time used variables really without strong reason.